### PR TITLE
Implement `TestE2E_Bridge_ChangeVotingPower`

### PR DESCRIPTION
**Pull request by Stefan-Ethernal**
# Description

Implemented `TestE2E_Bridge_ChangeVotingPower`, which stakes received awards for single validator and makes sure that `totalStake`/`votingPower` is changed accordingly and that block production and checkpoints submission still runs successfully, regardless of voting power change.

# Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [ ] Hotfix (change that solves an urgent issue, and requires immediate attention)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Checklist

- [x] I have assigned this PR to myself
- [x] I have added at least 1 reviewer
- [x] I have added the relevant labels
- [ ] I have updated the official documentation
- [ ] I have added sufficient documentation in code

## Testing

- [ ] I have tested this code with the official test suite
- [ ] I have tested this code manually

### CODE REVIEW

File: consensus/polybft/fsm.go [hunk 1]
```diff
@@ -504,6 +504,13 @@ func (f *fsm) getCurrentValidators(pendingBlockState *state.Transition) (Account
 		return nil, fmt.Errorf("failed to retrieve validator set for current block: %w", err)
 	}
 
+	var buf bytes.Buffer
```
Comment (goran-ethernal) [inline consensus/polybft/fsm.go]: Do we need this log? I suppose you added it for testing?

Comment (Stefan-Ethernal) [reply consensus/polybft/fsm.go]: Yes, it is pretty convenient for troubleshooting purposes. 

I've added conditional logging (only if debug logs are enabled, we are going to do this stuff):
https://github.com/0xPolygon/polygon-edge/pull/1075/commits/5630d99cd301202e3a0786e081613d864869e647

**Review by goran-ethernal — APPROVED** [review]
:rocket:

**Review by Nemanja0x — APPROVED** [review]

File: e2e-polybft/bridge_test.go [hunk 2]
```diff
@@ -370,6 +376,7 @@ func getExitProof(rpcAddress string, exitID, epoch, checkpointBlock uint64) (typ
 	return rspProof.Result, nil
 }
 
+// TODO: Remove this to some separate file, containing helper functions?
```
Comment (b00ris) [inline e2e-polybft/bridge_test.go:379]: 👍

**Review by b00ris — APPROVED** [review]
Good job!

**Pull request merged**